src/features/user_profile/components/SettingsForm/SettingsForm.module.css

```diff
@@ -77,7 +77,7 @@
   display: flex;
   flex-direction: column;
   max-width: var(--settingColumnWidth);
-  padding-bottom: 100%;
+  padding-bottom: 80%;
```

Comment:
I've just tried changing it to 40% and it seems to be working fine without having a completely empty screen.

Reply:
Okay no, 40% doesn't work in some cases, but 60% seems fine